content/en/docs/concepts/architecture/nodes.md

```diff
@@ -255,7 +255,7 @@ The node controller checks the state of each node every `--node-monitor-period`
 Heartbeats, sent by Kubernetes nodes, help determine the availability of a node.
 
 There are two forms of heartbeats: updates of `NodeStatus` and the
-[Lease object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#lease-v1-coordination-k8s-io).
+{{< api-reference page="cluster-resources/lease-v1" >}} object.
```

Comment:
How are we gonna differentiate the following definitions if we use this page naming style?

- gv: `admissionregistration.k8s.io/v1`, kind: `ServiceReference`
- gv: `apiextensions.k8s.io/v1`, kind: `ServiceReference`
- gv: `apiregistration.k8s.io/v1`, kind: `ServiceReference`

Reply:
These 3 definitions do not appear in the same page, they can be referenced in a different way:

- `gv: apiregistration.k8s.io/v1, kind: ServiceReference` can be referenced with
`{{< api-reference page="cluster-resources/api-service-v1" anchor="ServiceReference" >}}`
- `gv: apiextensions.k8s.io/v1, kind: ServiceReference` can be referenced with
`{{< api-reference page="extend-resources/custom-resource-definition-v1" anchor="ServiceReference" >}}`
- `gv: admissionregistration.k8s.io/v1, kind: ServiceReference` can bereferenced with:
`{{< api-reference page="extend-resources/validating-webhook-configuration-v1" anchor="ServiceReference" >}}`

Reply:
Those look unambiguous to me.